### ISSUE TITLE
Rotface - Vile Gas

### ISSUE THREAD

**Issue by bLeyaa** [WoWManiaUK/Redemption#4905]
Related to #4302 

**What is Happening:** So at the moment, Vile Gas is working fine in terms of the player who is being targeted by it. As in, taking damage with occasional immobilization, but being able to cast. However, if other player(s) are in 8 yards range of the initial hit target the same applies to them - which is wrong. 

**What Should happen:** Players in range of the initial hit target should be disoriented for 6 seconds, as well as anyone who gets in the way of the spit.

https://www.youtube.com/watch?v=ohXqiqV1ZcM

You can see that this caused a wipe as everyone was disoriented and taking damage from a single vile gas. On mania atm this would end up with people just taking dmg and nothing else.

**Comment by Rushor** [comment]
Do we have a proof for that?

**Comment by Rushor** [comment]
so what must bechanged here?

currently it works like that:

targeted player get's spell 72272.

that spell applies a dot effect, as well as spell: 69244 which ticks every 2 seconds and has also a dot effect and a SPELL_AURA_MOD_ROOT effect.

Effect of spell 69248 which ticks every 2 seconds and was linked to spell 72272 was removed in the old tickets. that spell 69248 applied a MECHANIC_DISORIENTED and SPELL_AURA_MOD_DECREASE_SPEED effect

**Comment by Rushor** [comment]
restored the old case.

please test and describe in detail next time before making the bug report. this requires more testing. thnak you.

**Comment by bLeyaa** [comment]
No, dont revert it, thats completely wrong. Players affected by initial Vile Gas should be able to cast.

I'll try to explain better:

Lets say Player A is standing alone and no one is around him - that player will have root effect every 2 sec and a dot damage. Player can continue casting the entire time.

Now let's say Player B is standing within 8 yards of Player A. The above example should still apply to Player A, as he was the initial target of it. However the Player B now should have the disorientation effect in addition to the rest of it. This is the whole purpose of being spread out.

Basically from that point on, anyone who links to Player B should be disoriented.

The only thing i am unsure of is if the Player A should get a disorientation effect IF the Player B is within the 8 yards, technically he should be if the Player B vomits on him, but i don't know how hard is it to make it work like that. 
If easier, just make it so that players who are alone don't get disoriented and in case they are stacked within 8 yards they do get the effect.

Hope this clears it up a bit.
